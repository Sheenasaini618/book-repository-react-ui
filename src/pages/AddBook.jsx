import GoogleBookCatalogue from "../components/GoogleBookCatalogue";
import NavbarTwo from "../components/NavbarTwo";
import { useState } from "react";
import React from "react";

function AddBook() {
  const [googleApiResult, setGoogleApiResult] = useState("");

  return (
    <div>
      <NavbarTwo setData={setGoogleApiResult} data={googleApiResult} />
      <GoogleBookCatalogue data={googleApiResult} />
    </div>
  );
}

export default AddBook;
