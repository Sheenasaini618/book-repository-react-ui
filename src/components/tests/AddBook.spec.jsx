import React from 'react';
import '@testing-library/jest-dom';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import AddBook from '../../pages/AddBook'

configure({ adapter: new Adapter() });

describe('<AddBook />', () => {

    let component;

    beforeEach(() => {

        component = shallow(<AddBook />);

    })

    it('should match snapshot', () => {
        expect(component.debug).toMatchSnapshot();
    });
})