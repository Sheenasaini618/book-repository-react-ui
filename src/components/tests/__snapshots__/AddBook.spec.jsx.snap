// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddBook /> should match snapshot 1`] = `[Function]`;
